src/commands/config/context/messages.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { ReactElement } from 'react';
+import { injectable } from 'tsyringe';
+import { Text, Box } from 'ink';
+import { Context } from '../../../config/context';
+import {
+  NO_CONTEXTS_SAVED,
+  CONTEXT_NOT_FOUND,
+  MISSING_CURRENT_CONTEXT,
+  NEW_CONTEXT_ADDED,
+  CONTEXT_DELETED
+} from '../../../messages';
+
+@injectable()
+export class Messages {
+  notContextSaved(): string {
+    return NO_CONTEXTS_SAVED;
+  }
+
+  contextNotFound(contextName: string): string {
+    return CONTEXT_NOT_FOUND(contextName);
+  }
+
+  missingCurrentContext(): string {
+    return MISSING_CURRENT_CONTEXT;
+  }
+
+  contextAdded(contextName: string) {
+    return NEW_CONTEXT_ADDED(contextName);
+  }
+
+  removeContext() {
+    return CONTEXT_DELETED;
+  }
+}
+
+@injectable()
+export class ContextMessageWriter {
+  list(ctx: Context): ReactElement {
+    return <Box flexDirection="column">
+      {Object.keys(ctx.store).map((contextName: string) => <Text key={contextName}>{contextName}</Text>)}
+    </Box>;
+  } 
+
+  current(ctx: Context): ReactElement {
+    return <Text>{ctx.current}</Text>;
+  }
+
+  add(message: string): ReactElement {
+    return <Text color="green">{message}</Text>;
+  }
+
+  use(ctx: Context): ReactElement {
+    return <Text>{ctx.current}: {ctx.getContext(ctx.current as string)}</Text>;
+  }
+
+  remove(message: string): ReactElement {
+    return <Text>{message}</Text>;
+  }
+
+  throwError(message: string) {
+    return <Text color="red">{message}</Text>;
+  }
```

Comment:
Why do we need this ContextMessageWritter?? I think that if we want to abstract or name any of the views, we can create a separate component but for me, this isn't so intuitive 🤔

Reply:
`ContextMessageWritter` might be overcomplicating things. Currently, I am just using it for running tests.

Reply:
I understand the key of having the text strings that will be displayed, not this case as far as I know (even could be intestesting to take care of future internacionalization) but extracting all of these set of texts in order to make easier the asserts on the tests, is not a good practice. You could extract individual views as components in order to make unit testing and make sure that those individual views behave as you expect. This could be useful if you only want to assert (on an integration test or bigger unit test that contains any of these single compontents) that it contains certain string, not to do an assert about all the view tree. 😛